docs/import-data/csv/load-csv-clause.md

```diff
@@ -1,11 +1,18 @@
 ---
 id: load-csv-clause
 title: LOAD CSV Cypher clause
-sidebar_label: LOAD CSV Cypher clause
+sidebar_label: CSV files
```

Comment:
CSV files, but JSON stayed the same?

Reply:
That page with JSON is interchangeably using format and file... so is it Loading from JSON file? Or format? Are files that have JSON immediately in JSON format... 

Maybe we should deal with that first, BCS I don't know should I make the title JSON files or JSON format